src/OmnichannelChatSDK.ts

```diff
@@ -154,6 +156,41 @@ class OmnichannelChatSDK {
         return this.liveChatConfig;
     }
 
+    private async updateChatToken(newToken: string, newRegionGTMS?: any): Promise<void> {
```

Comment:
We should add a new `TelemetryEvent` called `UpdateChatToken` add start the scenario at the beginning of the method call